packages/app/src/app/pages/Pro/index-v2.tsx

```diff
@@ -39,44 +41,168 @@ export const ProPage: React.FC = () => (
         <Route path={`/pro/create-workspace`}>
           <NewTeam redirectTo="/pro?v=2" />;
         </Route>
+        <Route path={'/pro/success'}>
+          <Success />
+        </Route>
         <Route path={`/pro`}>
-          <Upgrade />
+          <UpgradeSteps />
         </Route>
       </Switch>
     </Stack>
   </ThemeProvider>
 );
 
-
 const prettyPermissions = {
   ADMIN: 'Admin',
   WRITE: 'Editor',
   READ: 'Viewer',
 };
 
-const Upgrade = () => {
+type Plan = {
+  id: string;
+  name: string;
+  type: 'personal' | 'team';
+  frequency: 'monthly' | 'annual';
+  unit: number;
+  multiplier: number;
+  currency: string;
+};
+
+const PADDLE_VENDOR_ID = 729;
+const plans: { [key: string]: Plan } = {
```

Comment:
this _config_ is still a work in progress, will move it to constants when it's shape is frozen